sdk_experimental/lib/opentelemetry/sdk/trace/propagation/trace_context/response_text_map_propagator.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Trace
+    module Propagation
+      module TraceContext
+        # Propagates trace response using the W3C Trace Context format
+        # https://w3c.github.io/trace-context/#traceresponse-header
+        class ResponseTextMapPropagator
```

Comment:
Technically, you’ve implemented a TextMapInjector here rather than a propagator: https://github.com/open-telemetry/opentelemetry-ruby/blob/32f4a7a10ecce597a574a675fa797e29561c473d/api/lib/opentelemetry/context/propagation.rb#L31-L36

For a propagator, you’d also need to add a no-op extract implementation like https://github.com/open-telemetry/opentelemetry-ruby/blob/32f4a7a10ecce597a574a675fa797e29561c473d/api/lib/opentelemetry/context/propagation/noop_text_map_propagator.rb#L17-L19